app/schema.graphql

```diff
@@ -8,6 +8,19 @@ type Cluster {
   driftRatio: Float
 }
 
+"""Choose one (and only one) of the available options"""
+input ClustersFinder {
+  """
+  Hierarchical Density-Based Spatial Clustering of Applications with Noise: https://hdbscan.readthedocs.io/en/latest/index.html
+  """
+  hdbscan: HdbscanConfig
+
+  """
+  K-Means Clustering: https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html
+  """
+  kmeans: KmeansConfig
```

Comment:
Do we want to bite this off right now?

Reply:
this is just proof of concept, and mostly for testing the data structure, it's not intended to be a working implementation

Reply:
nvm I see your comment in the PR. Got me a bit scared there.